src/Http/Client.php

```diff
@@ -549,3 +549,5 @@ protected function _createAuth($auth, $options)
         return new $class($this, $options);
     }
 }
+// @deprecated Backwards compatibility with earler 3.x versions.
+class_alias('Cake\Http\Client', 'Cake\Network\Http\Client');
```

Comment:
Would this and other similar aliases referencing classes in `Cake\Networks` create problem if we want to provide `Cake\Http` as a standalone lib?